src/Microsoft.DotNet.XHarness.CLI/Microsoft.DotNet.XHarness.CLI.csproj

```diff
@@ -8,6 +8,22 @@
     <ToolCommandName>xharness</ToolCommandName>
     <!-- Mono.Options is apparently not strong-name signed -->
     <NoWarn>CS8002;</NoWarn>
+
+    <IsWindows Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">true</IsWindows>
```

Comment:
Suggestion: could more simply use the `$(OS)` property here or `[MSBuild]::IsOSPlatform()`